yarn-project/deploy_npm.sh

```diff
@@ -6,12 +6,14 @@ extract_repo yarn-project /usr/src project
 cd project/src/yarn-project
 
 echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc
+# also copy npcrc into the l1-contracts directory
+cp .npmrc ../l1-contracts
 
 function deploy_package() {
     REPOSITORY=$1
     cd $REPOSITORY
 
-    VERSION=$(extract_tag_version $REPOSITORY true)
+    VERSION=$(extract_tag_version $REPOSITORY false)
```

Comment:
Why is this one changed? Should it not error if failing anymore?

Reply:
The idea is to be able to take in this without a tag and see the entire process dry run. Open to thoughts here but this was changed by me then back by  @spalladino . This reverts to my intended change allowing dry runs

Reply:
So it should be `false`? @ludamad (remember I'm peanut-brain, so I just need a simple yes/no here 🥜).

Reply:
yea without it, you can't do dry runs

Reply:
Ahh good to know, I wasn't aware that that was the intention. Makes sense then!